test/harry/main/org/apache/cassandra/harry/SchemaSpec.java

```diff
@@ -182,7 +182,7 @@ public String compile()
             appendWith.run();
             if (shouldAppendAnd)
                 sb.append(" AND");
-            sb.append(" transactional_mode = '").append(options.transactionalMode().toString()).append('\'');
+            sb.append(" transactional_mode = '").append(options.transactionalMode().asCqlParam()).append('\'');
```

Comment:
commented in slack but this won't compile

```
this.cqlParam = String.format("transactional_mode = '%s'", this.name().toLowerCase());
```

this method handles the variable as well as the value... so you want

```
sb.append(" ").append(options.transactionalMode().asCqlParam());
```